app/Http/Controllers/V2/Dashboard/ProjectProfileDetailsController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\Dashboard\ProjectProfileDetailsResource;
+use App\Models\V2\Forms\FormOptionListOption;
+use App\Models\V2\Projects\Project;
+use Illuminate\Http\Request;
+
+class ProjectProfileDetailsController extends Controller
+{
+    public function __invoke(Request $request): ProjectProfileDetailsResource
+    {
+        if ($request->has('uuid')) {
```

Comment:
Since `uuid` is required to do anything useful in this endpoint, please include it in the URL instead of the query. The `api_v2` definition should be:
```
Route::get('/project-details/{project}', ProjectProfileDetailsController::class)
```

And then this method becomes 
```
public function __invoke(Request $request, Project $project): JsonResource
```

Laravel will know that you're seeking a Project instance with the UUID in the URL, get it out of the database and pass it directly into the method.